Site/package.json

```diff
@@ -26,7 +26,7 @@
     "@veupathdb/study-data-access": "^0.3.1",
     "@veupathdb/user-datasets": "^0.1.0",
     "@veupathdb/wdk-client": "^0.7.2",
-    "@veupathdb/web-common": "^0.5.2",
+    "@veupathdb/web-common": "file:local_modules/@veupathdb/web-common",
```

Comment:
You'll want to switch this out before merging, of course.

Reply:
Oops! Didn't mean to include these files